frontend/src/utils/formatDateTime.ts

```diff
@@ -0,0 +1,8 @@
+import type { YYYYMMDDHHMM } from '~/types/schedule';
+
+export const formatDateTime = (rawDateTime: YYYYMMDDHHMM) => {
+  const [rawDate, time] = rawDateTime.split(' ');
+  const date = rawDate.split('-');
+
+  return `${date[0]}년 ${date[1]}월 ${date[0]}일 ${time}`;
```

Comment:
```suggestion
  return `${date[0]}년 ${date[1]}월 ${date[2]}일 ${time}`;
```

날짜 부분이 올바르게 나오지 않는 것 같아~ 인덱스를 `2` 로 바꾸니까 올바르게 동작하는 데 어떤 것 같아?

![image](https://github.com/woowacourse-teams/2023-team-by-team/assets/87642422/87aae205-92c3-4492-bf0c-1c7931fc82c3)